### ISSUE TITLE
FEEDBACK: Read Only Image and Audio Questions

### ISSUE THREAD

**Issue by BrandonArmstrong** [tedrick/Survey123-Testing#159]
In version 3.1, image and audio questions are now able to be set as read only and required by 'yes' or an expression. 

Verify that this is working in the field app with new collection.

http://docdev.arcgis.com/en/survey123/desktop/create-surveys/xlsformexpressions.htm

**Comment by tedrick** [comment]
Sarah Bubka

**Comment by sbubka94** [comment]
I don't think this functionality is working. I included a sample mp3 file and an image in the media folder. Then I created 1 image and 1 audio question, assigning each their respective files in the media::image and media::audio columns. When I validate input (or attempt to send from published survey), it throws a message telling me those fields are required, despite having something present in those fields.

[Read Only Image Audio.xlsx](https://github.com/tedrick/Survey123-Testing/files/2447760/Read.Only.Image.Audio.xlsx)

![image](https://user-images.githubusercontent.com/22916127/46496822-d58fe300-c7cd-11e8-87ce-0a85932a181e.png)
